app/src/main/java/ch/epfl/skysync/models/location/Location.kt

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.skysync.models.location
+
+import ch.epfl.skysync.models.UNSET_ID
+import ch.epfl.skysync.models.user.User
+import com.google.android.gms.maps.model.LatLng
+
+/**
+ * Represents a Location
+ *
+ * @param user The user of whom we share the location
+ * @param value His actual location
+ */
+data class Location(
+    val id: String = UNSET_ID,
+    val user: User,

```

Comment:
I don't think you need to have a complete user here, `userId` is enough (It is defined that way in `Message` so that we can display the sender name)

Reply:
Makes it way simpler indeed